solr/core/src/java/org/apache/solr/cli/StatusTool.java

```diff
@@ -292,42 +291,26 @@ public Map<String, Object> waitToSeeSolrUp(
 
   public Map<String, Object> getStatus(String solrUrl, String credentials) throws Exception {
     try (var solrClient = CLIUtils.getSolrClient(solrUrl, credentials)) {
-      return getStatus(solrClient);
+      Map<String, Object> status = reportStatus(solrClient);
+      return status;
     }
   }
 
-  public Map<String, Object> getStatus(SolrClient solrClient) throws Exception {
-    Map<String, Object> status;
-
-    NamedList<Object> systemInfo =
-        solrClient.request(
-            new GenericSolrRequest(SolrRequest.METHOD.GET, CommonParams.SYSTEM_INFO_PATH));
-    // convert raw JSON into user-friendly output
-    status = reportStatus(systemInfo, solrClient);
-
-    return status;
-  }
-
-  public static Map<String, Object> reportStatus(NamedList<Object> info, SolrClient solrClient)
-      throws Exception {
+  public static Map<String, Object> reportStatus(SolrClient solrClient) throws Exception {
     Map<String, Object> status = new LinkedHashMap<>();
```

Comment:
Does having this status map help us?  If this map only ever has data sourced from SystemInfoResponse, a thought could be to just pass back a SystemInfoResponse object?   I don't know if that is a "bad" thing from the perspective that the http transport model object is bleeding into the app...   I hate passing maps of magic values around in a strongly type language like java!

Reply:
also, this could be fixed in a seperate PR after this to more clearly see the impact!